src/api/routers/categories.py

```diff
@@ -47,7 +47,7 @@ def create_category(category: CategoryCreate, session: SessionDep):
 
 @router.get("/", response_model=list[CategoryRead])
 def read_categories(session: SessionDep):
-    categories = session.exec(select(Category)).all()
+    categories = session.exec(select(Category).order_by(Category.name)).all()
```

Comment:
ok, FYI with Django you can set default ordering on the model with `Meta.ordering`